api/core/domain/daily_user_digest.py

```diff
@@ -0,0 +1,31 @@
+from datetime import date
+
+from pydantic import BaseModel
+
+
+class DailyUserDigest(BaseModel):
```

Comment:
Use a `NamedTuple` to avoid the overhead ? 
Also move the the customer service since it's only used there ? `domain` layer is mostly fo cross layer communication ?

Reply:
moved https://github.com/WorkflowAI/WorkflowAI/pull/161/commits/cc4ed104aa9905d7b48f4962c887c9862799f011